code/datums/proximity_monitor/fields/heretic_arena.dm

```diff
@@ -0,0 +1,267 @@
+		return
+	if(!isobj(attacking_item))
+		return
+	var/obj/attacking_object = attacking_item
+
+	// Track being hit by a mob holding a stick
+	if(ismob(attacking_object.loc))
+		last_attacker = attacking_object.loc
+		return
+
+	// Track being hit by a mob throwing a stick
+	if(isitem(attacking_object))
+		var/obj/item/thrown_item = attacking_item
+		var/thrown_by = thrown_item.thrownby?.resolve()
+		if(ismob(thrown_by))
+			last_attacker = thrown_by
+			return
+
+	// Edge case. If our attacking_item is a gun which the owner has dropped we need to find out who shot us
+	// Track being hit by a mob shooting a stick
+	if(isprojectile(attacking_object))
+		var/obj/projectile/attacking_projectile = attacking_object
+		if(ismob(attacking_projectile.firer))
+			last_attacker = attacking_projectile.firer
+
+/// Called when you crit somebody to update your crown
+/datum/status_effect/arena_tracker/proc/on_crit_somebody()
+	SIGNAL_HANDLER
+	owner.cut_overlay(crown_overlay)
+	crown_overlay = mutable_appearance('icons/mob/effects/crown.dmi', "arena_victor", -HALO_LAYER)
+	crown_overlay.pixel_y = 24
+	owner.add_overlay(crown_overlay)
+	owner.remove_traits(list(TRAIT_ELDRITCH_ARENA_PARTICIPANT, TRAIT_NO_TELEPORT), STATUS_EFFECT_TRAIT)
+
+	// The mansus celebrates your efforts
+	if(IS_HERETIC(owner))
+		owner.heal_overall_damage(60, 60, 60)
+		owner.adjustToxLoss(-60)
+		owner.adjustOxyLoss(-60)
+		if(iscarbon(owner))
+			var/mob/living/carbon/carbon_owner = owner
+			for(var/datum/wound/wound as anything in carbon_owner.all_wounds)
+				wound.remove_wound()
+
+	if(arena_victor) // No need to spam if we've already killed at least 1 person
+		return
+	if(IS_HERETIC(owner))
+		to_chat(owner, span_big(span_hypnophrase("The mansus is pleased with your performance, you may leave now.")))
+	else
+		to_chat(owner, span_big(span_hypnophrase("You have done well, you may leave now.")))
+	arena_victor = TRUE
+
+/datum/antagonist/heretic_arena_participant
+	name = "Arena Participant"
+	show_in_roundend = FALSE
+	replace_banned = FALSE
+	objectives = list()
+	antag_hud_name = "brainwashed"
+	block_midrounds = FALSE
+
+/datum/antagonist/heretic_arena_participant/on_gain()
+	forge_objectives()
+	. = ..()
+
+/datum/antagonist/heretic_arena_participant/forge_objectives()
+	var/datum/objective/survive = new /datum/objective
+	survive.owner = owner
+	survive.explanation_text = "You are trapped in a battlefield where each participant must fight to the death. \
+		Defeat your captor, or betray your fellows so you may live to see another day!"
+	objectives += survive
+	var/datum/objective/fight_to_escape = new /datum/objective
+	fight_to_escape.owner = owner
+	fight_to_escape.explanation_text = "DO NOT ATTEMPT TO ESCAPE. The only way out is to defeat another participant in a battle to the death. \
+		A pitiful blade has been given to you so you may have a fighting chance, it would be a shame were you to ever break it while unworthy."
```

Comment:
```suggestion
	survive.explanation_text = "You have been trapped in an arena. The only way out is to slaughter someone else. Kill your captor, or betray your friends - the choice is yours."
	objectives += survive
	var/datum/objective/fight_to_escape = new /datum/objective
	fight_to_escape.owner = owner
	fight_to_escape.explanation_text = "Escape is <b>impossible</b>. The only way out is to defeat another participant in this battle to the death. \
		A weapon has been bestowed unto you, granting you a fighting chance, it would be quite a shame were you to attempt to break it."
```
The "DO NOT ATTEMPT TO ESCAPE" comes off as a bit too much.

Reply:
idk why it's failing to close the quotes properly here